memdown.js

```diff
@@ -136,6 +136,7 @@ MemDOWN.prototype._open = function (options, callback) {
 }
 
 MemDOWN.prototype._put = function (key, value, options, callback) {
+  if (typeof value === 'undefined' || value === null) value = ''
```

Comment:
what about a `false` value? aka could this be generalized to `value = value || ''`